src/main/webapp/app/tasks/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { Task } from "./task";
+
+export function sortTasksByStatus(tasks: Task[]) {
```

Comment:
missing return type definition

Reply:
I think I will definitely remember to add return types to all functions from now on :)